### DISCUSSION TITLE
How to create another site, under one CMS

### DISCUSSION THREAD

**Discussion by webmedia1012** [OrchardCMS/OrchardCore#10280] [Q&A] [answered]
Hi All, 
I wanted to create another site, without creating a new CMS project (and possibly a new SQL) how do I go about doing it? I see on app_data/sites/default where I can potentially have app_data/sites/site2/.

Thank you so much in advanced.

**Comment by ns8482e** [comment, marked as answer]
Enable Tenants feature

**Reply by webmedia1012** [reply]
Then ? do I just setup a site? and is that on the feature? - thanks!
